annotation-processing/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
gradle wrapper isn't needed in child project. remove everything that relates to it